src/i18n/content/jp/docs/apis/nerdgraph/examples/nerdgraph-historical-data-export.mdx

```diff
@@ -9,7 +9,7 @@ freshnessValidatedDate: never
 translationType: machine
 ---
 
-[NRQLクエリ](/docs/query-your-data/nrql-new-relic-query-language/get-started/introduction-nrql-new-relics-query-language)を実行する場合、応答で返されるデータポイントの数の制限やクエリのタイムアウトなど、 [さまざまな](/docs/query-your-data/nrql-new-relic-query-language/get-started/rate-limits-nrql-queries)制限によって制限されます。履歴データのエクスポート機能を使用して、応答で200,000,000データポイントを返し（通常の[制限である2000](/docs/query-your-data/nrql-new-relic-query-language/get-started/nrql-syntax-clauses-functions/#sel-limit)ではなく）、クエリのタイムアウトがないNRQLクエリを実行できます。結果は1つ以上のJSONファイルとして返されます。
```

Comment:
Do we actually need to update these ourselves or should they automatically generated?

Reply:
We do not need to update these. We should undo this commit, as it will create build issues for this PR